neurosynth/analysis/reduce.py

```diff
@@ -226,17 +227,18 @@ def topic_models(abstracts, n_topics=50, n_words=31, n_iters=1000, alpha=None,
             abstract = abstracts.loc[pmid]['abstract']
             with open(os.path.join(absdir, str(pmid)+'.txt'), 'w') as fo:
                 fo.write(abstract)
-        
+
     # Run MALLET topic modeling
     print('Generating topics...')
+    mallet_bin = abspath('../resources/mallet/bin/mallet')
```

Comment:
Seems like this will fail if the user manually changes directory. Maybe grab the module directory with something like `os.path.join(dirname(__file__), ... 'resources')`?